packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/Models/Friend.Serialization.cs

```diff
@@ -64,6 +65,33 @@ Friend IJsonModel<Friend>.Create(ref Utf8JsonReader reader, ModelReaderWriterOpt
             throw new NotImplementedException("Not implemented");
         }
 
+        internal static Friend DeserializeFriend(JsonElement element, ModelReaderWriterOptions options = null)
```

Comment:
This method was a hold over from the IUtf8JsonSerializable days.  This body is the same as what we need but this will be the CreateCore method implementation that IJsonModel.Create calls